### PR TITLE
Italian Alexa Update

### PR DESCRIPTION
**Pull request by francesc0-cgm**
All Alexa answers translation in Italian

### CODE REVIEW

**Comment by francesc0-cgm** [comment]
Added other translations

**Comment by PieterGit** [comment]
@francesc0-cgm We need to see why this PR failed on Node 10 (and works on Node 8). Just restarted the build. Hoping @sulkaharo can have a look at it too.

@francesc0-cgm I also would like a different Alexa user to confirm these changes work and don't cause problems for non-Italian Alexa users. Do you know of another Italian and non Italian user that can test?

**Comment by PieterGit** [comment]
@francesc0-cgm Please also have a look at the Codacy error:
https://app.codacy.com/app/Nightscout/cgm-remote-monitor/pullRequest?prid=2472536&bid=9517397
Error `'ctx' is defined but never used. (no-unused-vars)` in `lib/plugins/ar2.js`

**Comment by PieterGit** [comment]
Ok, restarting the Node 10 build fixed the problem, so it's some kind of Travis weirdness. If the Codacy error is resolved and if we can find a confirmation that it works or does not regresses stuff for non italian users, I will merge the pull request.

**Comment by francesc0-cgm** [comment]
I confirm that worked for another italian girl, except for loop prediction that is still in english. For other languages we should wait someone else

**Comment by androckz** [comment]
Hi @francesc0-cgm I could help you testing this PR.
Where can I find the .json for the italian skill?
Thank you

**Comment by francesc0-cgm** [comment]
Here it is 
https://api.myjson.com/bins/1dgady

Inviato da iPhone

**Comment by PieterGit** [comment]
@francesc0-cgm I moved the `lib/plugins/alexa-plugin.md` to `docs/plugins`

I discussed with @sulkaharo and the Alexa plugin should be more loosely coupled from the other files. I hope @sulkaharo can review this and that @francesc0-cgm and perhaps other Alexa users/developers can fix this for the Nightscout 0.11 update.

**Comment by francesc0-cgm** [comment]
I think docs is clear and easy to apply @pietergit 
I think to leave translation to every NS international group

File: lib/plugins/ar2.js [hunk 1]
```diff
@@ -145,7 +147,7 @@ function init (ctx) {
     return result.points;
   };
 
-  function alexaAr2Handler (next, slots, sbx) {
+  function alexaAr2Handler(next, slots, sbx) {
```
Comment (PieterGit) [inline lib/plugins/ar2.js]: The ar2.js should not contain any alexa specific code in it.

File: lib/plugins/cob.js [hunk 2]
```diff
@@ -292,21 +294,32 @@ function init(ctx) {
     });
   };
 
-  function alexaCOBHandler (next, slots, sbx) {
-    var preamble = (slots && slots.pwd && slots.pwd.value) ? slots.pwd.value.replace('\'s', '') + ' has' : 'You have';
-    var value = 'no';
+  function alexaCOBHandler(next, slots, sbx) {
```
Comment (PieterGit) [inline lib/plugins/cob.js]: this should not be in the basic COB code, but be abstracted in the alexa plugin

**Comment by PieterGit** [comment]
@francesc0-cgm Please check https://github.com/nightscout/cgm-remote-monitor/pull/4980 and help test that Alexa integration. If this PR can be reimplemented in a generic way please rewrite the PR to do so. Closing this PR due to merge conflicts and  out of date.

**Comment by inventor96** [comment]
@francesc0-cgm Your translation work would be very helpful! #4980 provides instructions on how to add additional language support in the way the virtual assistants are built to handle it.

**Pull request closed without merging**